app/views/forms/submission_email/submission_email_code_sent.html.erb

```diff
@@ -1,8 +1,8 @@
-<h1 class="govuk-heading-l">
-  <span class="govuk-caption-l"><%=  @submission_email_form.form.name %></span>
+<h1 class="govuk-heading-l" data-notification-uuid="<%= @submission_email_form.notify_response_id %>">
```

Comment:
What's this data attribute for? I don't think it's read elsewhere in this PR or in the e2e tests